boards/itsybitsy_m4/examples/spi.rs

```diff
@@ -0,0 +1,73 @@
+#![no_std]
+#![no_main]
+
+/// This example shows a simple transfer operation with an slave device.
+/// The ItsyBitsy will send a simple Hello World message, and the slave
+/// is expected to send a response. After the transaction, the response
+/// from the slave is echoed in the default UART.
+
+#[cfg(not(feature = "use_semihosting"))]
+use panic_halt as _;
+
+#[cfg(feature = "use_semihosting")]
+use panic_semihosting as _;
+
+use itsybitsy_m4 as bsp;
+
+use atsamd_hal::{delay::Delay, time::MegaHertz};
+use bsp::{
+    entry,
+    hal::{
+        clock::GenericClockController,
+        pac::{CorePeripherals, Peripherals},
+        prelude::*,
+    },
+    spi_master, Hertz,
+};
+#[entry]
```

Comment:
I think there's usually two lines between `use` statements and functions. I'm surprised our formatting CI didn't flag this... 🤔

Reply:
ah just checked, apparently there is no whitespace separation requirement here. I personally prefer one or two lines but 🤷🏼